monitoring/overlays/cnv-prod/alerting-rules.yaml

```diff
@@ -0,0 +1,140 @@
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="investigator"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Investigator is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdGraphSync
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="graph-sync"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Graph Sync is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdWorkflowHelpers
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="workflow-helpers"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Workflow Helpers is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdMetricsExporter
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="metrics-exporter"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Metrics Exporter is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdManagementAPI
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="management-api"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Management API is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdAdviseReporter
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="advise-reporter"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Advise Reporter is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdGraphBackup
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="graph-backup-job"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Graph Backup is not in sync with database schema."
+            severity: "HIGH"
+
+        - alert: ThothSchemaCheckProdPackageReleases
+          expr: thoth_graph_db_component_revision_check{env="zero-prod", component="thoth-package-release-job"} == 1
+          for: 5m
+          annotations:
+            summary: "Schema of Package Release is not in sync with database schema."
+            severity: "HIGH"
+
+          # Alert for amcheck database corruption check.
+        - alert: ThothDatabaseCorruptionCheckProd
+          expr: thoth_graphdb_is_corrupted{instance="metrics-exporter-thoth-infra-prod.apps.zero.massopen.cloud:80"} == 1
+          for: 5m
+          annotations:
+            summary: "Database schema is corrupted, all services need to be stopped."
+            severity: "HIGH"
+
+          # Alert for alembic version mismatch between components and database.
```

Comment:
# Alert for alembic version table corruption

Reply:
Thanks :+1: